src/Participant.hpp

```diff
@@ -98,7 +99,10 @@ class Participant
      * @throws DDSMiddlewareException If the *DomainParticipant* could not be created.
      */
     void build_participant(
-            const ::fastdds::dds::DomainId_t& domain_id = 0);
+            const YAML::Node& config);
+
+    //! Build participant with empty configuration
```

Comment:
Comments should follow same doxygen style as the whole file does

Reply:
:+1:

Reply:
Still unresolved